pkg/lib/serverTime.js

```diff
@@ -321,13 +313,131 @@
                     console.warn("failed to load time servers", error);
                     return result;
                 });
-    };
+    }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17992-20230110-094617-6e1ebb8f-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)